app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -766,11 +766,15 @@ def build_toolbar_hide_button(id)
     when "Service"
       return build_toolbar_hide_button_service(id)
     when "Vm"
+      # maybe use .active? instead of .orphaned?
+      # vm_or_template.rb #1331
       case id
       when "vm_clone"
         return true unless @record.cloneable?
+        return true if @record.orphaned?
```

Comment:
Are orphaned VMs not cloneable?  If so, the `cloneable?` method should be taught that - and not extra logic added here.